readme.md

```diff
@@ -4,8 +4,26 @@
 

```

Comment:
Does it make sense to have a little blurb at the top of the readme just generally describing what this project is and it's purpose? I realize no one will probably be finding themselves here for awhile but, might be nice.

Reply:
Yes, never hurts! I'll put this on my list for updating in this PR.